languages/typescript/src/main/kotlin/io/vrap/codegen/languages/typescript/server/ServerRenderer.kt

```diff
@@ -230,7 +230,7 @@ class ServerRenderer @Inject constructor(
         return when (this) {
             is VrapObjectType -> {
                 val joiType = this.toJoiVrapType()
-                "import { ${joiType.simpleClassName} } from '@joi/${joiType.`package`}'"
+                "import { ${joiType.simpleClassName} } from '${joiType.`package`}'"
```

Comment:
Actually this was done on purpose to allow the separation of joi package from the model package, usually they might be the same thing but not guaranteed 😢

Reply:
In the server generator we now put the generated joi files in the same folder as our model files. And this then makes it difficult to add another path here. We just tried to be a bit more pragmatic here 😉

Reply:
Sounds good to me, to be fair i don't separate them anymore anywhere. so this wont break any of my work anywhere 👍